src/clients/go/pkg/types/main.go

```diff
@@ -82,7 +82,7 @@ func HexStringToUint128(value string) (Uint128, error) {
 }
 
 // BigIntToUint128 converts a [math/big.Int] to a Uint128.
-func BigIntToUint128(value big.Int) Uint128 {
+func BigIntToUint128(value *big.Int) Uint128 {
```

Comment:
Interacting with `*big.Int` pointers is indeed more convenient, so this change is nice to keep.

Reply:
If bytes works the same I think in this case would be basically the same usability wise.